elas:
  disp_min: 0
  disp_max: 255
  support_texture: 12
  candidate_stepsize: 20
  incon_window_size: 3
  incon_threshold: 2
  incon_min_support: 30
  grid_size: 10
  support_threshold: 0.85
  beta: 0.26
  gamma: 55
  sigma: 2
  sradius: 2
  speckle_sim_threshold: 40
  match_texture: 26
  lr_threshold: 7
  speckle_size: 1021
  ipol_gap_width: 10
  add_corners: True
  filter_median: True
  filter_adaptive_mean: True
  postprocess_only_left: True
  subsampling: False
